department-of-veterans-affairs/abd-vro
# MCP-2316: Change HTTP status to 400

## What was the problem?

We were getting 500 HTTP status for out of scope diagnostic codes for v2/health-data-assessment

Associated tickets or Slack threads:

- [MCP-2316](https://amida.atlassian.net/browse/MCP-2316)

## How does this fix it?[^1]

Change HTTP status to 400

## How to test this PR
- Step 1 build and start app, try v2/health-data-assessment with out of scope diagnostic code
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
